cli/options/src/lib.rs

```diff
@@ -113,10 +113,23 @@ impl NormalizePaths for PathOrDash {
     }
 }
 
+#[derive(JsonSchema, Parser, Debug, Clone, Serialize, Deserialize)]
+pub struct FStarOptions {
```

Comment:
We don't need to add a full test yet, but can you add an example in the documentation at least so people know how to use it?